### PR TITLE
Switched Explorer args to property to get it to persist in Settings.json

### PR DESCRIPTION
**Pull request by HobbyG**
Small change to get the custom parameters to persist in the settings.

### CODE REVIEW

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
+}
```
Comment (jjw24) [inline .vs/VSWorkspaceState.json]: this is not needed right?

Comment (HobbyG) [reply .vs/VSWorkspaceState.json]: No, it isn't needed, sorry about that.

Comment (jjw24) [reply .vs/VSWorkspaceState.json]: all good. Let me know when you removed it, I will approve it. LGTM. Thank you for the change

**Comment by jjw24** [comment]
Please version bump in plugin.json

File: .vs/VSWorkspaceState.json (hunk 1 above)
Comment (HobbyG) [reply .vs/VSWorkspaceState.json]: Bumped the version to 1.4.9 and deleted the needless file.

**Review by jjw24 — APPROVED** [review]

**Pull request merged**